### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by malvikasharan**
## Background

- Blog post on creating an OBF wide code of conduct https://www.open-bio.org/2019/11/15/creating-an-obf-wide-code-of-conduct/
- Blog post requesting comments https://www.open-bio.org/2021/05/12/request-for-comments-on-coc-draft/
- Board meeting where board agreed to be bound by membership vote on this PR https://www.open-bio.org/2021/09/11/obf-public-board-meeting-2021-09-21/

## Summary

- Upon agreement with the OBF board, a CoC is being proposed for OBF.
- An enforcement manual and reporting guideline will be added subsequently.
- The comments have been accumulated on issue #67 
- A BoF at BCC2020 will share this doc with the community

## Reference document:

The Carpentries CoC: https://docs.carpentries.org/topic_folders/policies/index_coc.html

### CODE REVIEW

File: CODE_OF_CONDUCT.md
Comment (yochannah) [inline CODE_OF_CONDUCT.md:31]:
```suggestion
Until the dedicated committee and contact details are created, you can reach out to the following members of the OBF board: Bastian Greshake (email: bgreshake@googlemail.com), Malvika Sharan (email: malvikasharan@gmail.com) and/or Yo Yehudi (yochannah@gmail.com).
```

**Comment by nlharris** [comment]
Is this ready to merge?

**Review by cjfields — APPROVED** [review]
Looks great!

**Comment by nlharris** [comment]
I'm satisfied with the substance of the document. I made some editorial suggestions in the gdoc.

**Review by nlharris — APPROVED** [review]
I'm satisfied with the substance of the document. I made some editorial suggestions in the gdoc.

**Comment by cjfields** [comment]
@peterjc @hlapp @gedankenstuecke have a look when you can. We're on the cusp! (I know @yochannah is already looking 😉)

**Review by yochannah — APPROVED** [review]

**Review by gedankenstuecke — APPROVED** [review]
As @nlharris: Happy with the substance, the language-edits in the Google Doc look good to me too!

**Review by HLWiencko — APPROVED** [review]

**Comment by cjfields** [comment]
This is ready to go, thanks everyone! @peterjc does this need public board approval prior to merging?

**Comment by hlapp** [comment]
> @peterjc does this need public board approval prior to merging?

I would think so. Also, we said this needs a membership vote in approval.

**Comment by cjfields** [comment]
@hlapp okay, will add that this needs a membership vote.

**Comment by cjfields** [comment]
We can merge this with 5 approvals, but will block until approval via membership vote

**Comment by malvikasharan** [comment]
@nlharris @cjfields @hlapp @gedankenstuecke @yochannah @HLWiencko This PR has been updated with all the documents. We need to add the dedicated emails in all the documents including for ombudsperson.

@tracykteal I hope you can have a look at the documents and recommend any wording or procedure that should be updated.

**Comment by cjfields** [comment]
> We can merge this with 5 approvals, but will block until approval via membership vote

Nevermind, will add a disclaimer that this is provisional and needs acceptance by the membership to be official

File: code-of-conduct/CODE_OF_CONDUCT.md
Comment (hlapp) [inline code-of-conduct/CODE_OF_CONDUCT.md:60]: This seems redundant with the Overview section. Does this really need to be in both places?

File: code-of-conduct/CODE_OF_CONDUCT.md
Comment (hlapp) [inline code-of-conduct/CODE_OF_CONDUCT.md:88]:
```suggestion
Participants who are asked to stop any inappropriate behavior are expected to comply immediately. This applies to any OBF events and platforms, either online or in-person. If a participant engages in behavior that violates this Code of Conduct, the organizers may warn the offender, ask them to leave the event or platform (without refund), or engage the Code of Conduct Committee to investigate the Code of Conduct violation and impose appropriate sanctions as described in the reporting and report-handling guidelines.
```

File: code-of-conduct/enforcement-manual.md
Comment (hlapp) [inline code-of-conduct/enforcement-manual.md:38]:
```suggestion
If the committee cannot reach consensus and deadlocks for over a week, the appointed members from the OBF Board will break the tie.
```

Comment (hlapp) [reply code-of-conduct/enforcement-manual.md:38]: Having _consensus_, _deadlock_, and _tie_ appear in the same sentence in this way is confusing. Does _deadlock_ mean one of the committee members refuses to join what is the consensus of the other committee members (in which case there is no tie that needs to be broken), or does deadlock mean there is a tie that needs to be broken (in which case there is no consensus, and tie-breaking won't produce a consensus).

File: code-of-conduct/enforcement-manual.md
Comment (hlapp) [inline code-of-conduct/enforcement-manual.md:39]:
```suggestion
If these members are unable to take part in the discussion due to a conflict of interest, the President of the OBF Board will make the decision.
```

File: code-of-conduct/enforcement-manual.md
Comment (hlapp) [inline code-of-conduct/enforcement-manual.md:49]: What does _research team_ mean?

File: code-of-conduct/incident-reporting-guidelines.md
Comment (hlapp) [inline code-of-conduct/incident-reporting-guidelines.md:10]:
```suggestion
**Ombudsperson**: If you are uncomfortable emailing the OBF's Code of Conduct Committee, you can send your incident report to the ombudsperson. The designated ombudsperson for OBF is **Tracy Teal**, who can be contacted by emailing `ombuds@open-bio.org`. Reports made to the ombudsperson will not be shared with the committee unless agreed with the reporters. 
```

File: code-of-conduct/incident-reporting-guidelines.md
Comment (hlapp) [inline code-of-conduct/incident-reporting-guidelines.md:5]:
```suggestion
Reports to the Code of Conduct Committee should be made by emailing `coc@open-bio.org`. All reports will be reviewed by the Code of Conduct Committee and will be kept confidential.
```

File: code-of-conduct/README.md
Comment (hlapp) [inline code-of-conduct/README.md:15]:
```suggestion
- Code of Conduct Committee: `coc@open-bio.org`
```

File: code-of-conduct/README.md
Comment (hlapp) [inline code-of-conduct/README.md:16]:
```suggestion
- Ombudsperson: `ombuds@open-bio.org`
```

File: code-of-conduct/CODE_OF_CONDUCT.md
Comment (hlapp) [inline code-of-conduct/CODE_OF_CONDUCT.md:48]:
```suggestion
Reports to the Code of Conduct Committee should be made by emailing `coc@open-bio.org`. Currently appointed members to report any incident of violation of Code of Conduct are Bastian Greshake Tzovaras, Malvika Sharan and Yo Yehudi. They are also available to respond to any question and concerns regarding the Code of Conduct document and procedure.
```

File: code-of-conduct/CODE_OF_CONDUCT.md
Comment (hlapp) [inline code-of-conduct/CODE_OF_CONDUCT.md:15]:
```suggestion
Please contact the Code of Conduct Committee by emailing `coc@open-bio.org`.
```

**Review by hlapp — COMMENTED** [review]
See inline comments, suggestions, and questions.

File: code-of-conduct/CODE_OF_CONDUCT.md
Comment (malvikasharan) [reply code-of-conduct/CODE_OF_CONDUCT.md:22]: We took that directly from the reference, but I think it is better to remove that as I can't explain what it really means,
.

File: code-of-conduct/CODE_OF_CONDUCT.md
Comment (malvikasharan) [reply code-of-conduct/CODE_OF_CONDUCT.md:28]: Removed.

File: code-of-conduct/CODE_OF_CONDUCT.md
Comment (malvikasharan) [inline code-of-conduct/CODE_OF_CONDUCT.md:53]:
```suggestion
All participants in our events and communications are expected to show respect and courtesy to others. All interactions should be professional regardless of platform: either online or in-person.
```

File: code-of-conduct/CODE_OF_CONDUCT.md
Comment (malvikasharan) [reply code-of-conduct/CODE_OF_CONDUCT.md:59]: No. I can edit that.

File: code-of-conduct/enforcement-manual.md
Comment (malvikasharan) [reply code-of-conduct/enforcement-manual.md:49]: That must be an error, updating to coc committee

**Review by hlapp — APPROVED** [review]

**Comment by brianrepko** [comment]
The section "Communication and community engagement platforms" might need some editing. We may want to update the Gitter channel reference (first one) or remove that as we refer to Gitter, Slack, and Twitter at the end of that sentence.

Otherwise read through everything and 👍

File: code-of-conduct/CODE_OF_CONDUCT.md
Comment (malvikasharan) [inline code-of-conduct/CODE_OF_CONDUCT.md:72]:
```suggestion
**Communication and community engagement platforms**, such as project repositories on GitHub (issues, PRs or other contributions), mailing list, website (blogs and comment sections), workshops or training events, and communication platforms such as Gitter, Slack and Twitter.
```

Comment (malvikasharan) [reply code-of-conduct/CODE_OF_CONDUCT.md:72]: @brianrepko can you please double-check this edit?

**Review by heuermh — COMMENTED** [review]
Thank you to all!

**Review by heuermh — APPROVED** [review]

**Comment by kipkurui** [comment]
Thank you for a clear Code of Conduct.  I may have missed it, but I did not see the avenues for appeal when a resolution has been made. Is that captured somewhere?

**Comment by malvikasharan** [comment]
Hi Caleb, you're right. We have not added the appeal process, which definitely should be integrated.
We can in principle take The Carpenries process as described here: https://docs.carpentries.org/topic_folders/policies/enforcement-guidelines.html.

**Comment by malvikasharan** [comment]
I have added an appeal process in the enforcement manual.

**Comment by kipkurui** [comment]
> I have added an appeal process in the enforcement manual.

Thanks, Malviak. Looks good!

**Review by nlharris — APPROVED** [review]

**Comment by cjfields** [comment]
@peterjc just a friendly reminder, we'll need your review on this one

**Comment by peterjc** [comment]
Note: We would need to update https://www.open-bio.org/code-of-conduct/ after merging this PR.

**Comment by nlharris** [comment]
@peterjc You haven't done your official review of this yet.

File: code-of-conduct/enforcement-manual.md
Comment (peterjc) [inline code-of-conduct/enforcement-manual.md:58]: I don't believe this needs adding to the main bylaws under the responsibilities of the officer roles. i.e. No change needed.

**Review by peterjc — APPROVED** [review]

File: code-of-conduct/enforcement-manual.md
Comment (hlapp) [reply code-of-conduct/enforcement-manual.md:58]: I would agree.

**Comment by peterjc** [comment]
Approved by balloting the OBF members, merging: https://vote.heliosvoting.org/helios/e/obf-coc-css

**Pull request merged**
